tests/requirements.txt

```diff
@@ -1,4 +1,5 @@
 pytest
+coverage<5.0
```

Comment:
I don't recall whether you can add comments to this file, but if you can it would be worth adding a comment as to why this pin is here so that it can be relaxed later when an issue if fixed.